client/login/wp-login/index.jsx

```diff
@@ -81,10 +83,10 @@ export class Login extends React.Component {
 		this.props.recordPageView( url, title );
 	}
 
-	renderLocaleSuggestions() {
-		const { locale, path, twoFactorAuthType, socialConnect } = this.props;
+	renderI18nSuggestions() {
+		const { locale, path, isLoginView } = this.props;
 
-		if ( twoFactorAuthType || socialConnect ) {
+		if ( ! isLoginView ) {
```

Comment:
Since `isLoginView` === `! twoFactorAuthType || ! socialConnect`, `! isLoginView` should equal to `twoFactorAuthType && socialConnect`, which deviates from the original expression.

Reply:
I didn't name that variable very well. 😞 

`isLoginView` was meant to convey: _this is the regular login landing view_. I only want to show the translator nudges on that view, and now I see the OR operand is totally useless :) Thanks for seeing that.

I'll change the the condition to be something more like: `! twoFactorAuthType && ! socialConnect`

Anything that would make it clearer do you think? I did think about flipping it and testing for `isLoginPageVariation` (is it anything but the login view).

Reply:
`isLoginView` reads quite precise to me, so I'm with you here :)